packages/resources-manager/src/browser/resources/tw.tsx

```diff
@@ -12,12 +12,16 @@ export const twResource: ScribeResource<Door43RepoResponse, TranslationWord[]> =
     id: "codex.tw",
     displayLabel: "Words",
     icon: <IconLink />,
-    getTableDisplayData: async () => {
+    getTableDisplayData: async (query?: string) => {
       try {
-        const data = await fetchDoor43ResourceDisplayData(twResource.id, {
-          subject: "Translation Words",
-          metadataType: "rc",
-        });
+        const data = await fetchDoor43ResourceDisplayData(
+          twResource.id,
+          {
+            subject: "Translation Words",
+            metadataType: "rc",
+          },
+          query
```

Comment:
Same here just merge it into the object as it's much cleaner and allows more flexibility going forwards.